src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/summary/MethodologySummaryEditPage.tsx

```diff
@@ -1,60 +1,66 @@
-import MethodologySummaryForm from '@admin/pages/methodology/components/MethodologySummaryForm';
+import MethodologySummaryForm, {
+  MethodologySummaryFormValues,
+} from '@admin/pages/methodology/components/MethodologySummaryForm';
 import {
   MethodologyRouteParams,
   methodologySummaryRoute,
 } from '@admin/routes/methodologyRoutes';
 import methodologyService from '@admin/services/methodologyService';
-import LoadingSpinner from '@common/components/LoadingSpinner';
-import useAsyncHandledRetry from '@common/hooks/useAsyncHandledRetry';
+import { useMethodologyContext } from '@admin/pages/methodology/contexts/MethodologyContext';
 import React from 'react';
 import { generatePath, RouteComponentProps } from 'react-router';
 
 const MethodologySummaryEditPage = ({
   history,
-  match,
 }: RouteComponentProps<MethodologyRouteParams>) => {
-  const { methodologyId } = match.params;
+  const {
+    methodologyId,
+    methodology,
+    onMethodologyChange,
+  } = useMethodologyContext();
 
-  const { value: methodology, isLoading } = useAsyncHandledRetry(async () => {
-    return methodologyService.getMethodology(methodologyId);
-  });
+  const handleSubmit = async (values: MethodologySummaryFormValues) => {
+    if (!methodology) {
+      throw new Error('Could not update missing methodology');
+    }
+
+    const nextMethodology = await methodologyService.updateMethodology(
+      methodologyId,
+      {
+        ...methodology,
+        title: values.title,
+      },
+    );
```

Comment:
I suspect this is going to cause a merge conflict with Ben's imminent PR: https://github.com/dfe-analytical-services/explore-education-statistics/pull/2785

Reply:
Yep! Rebased and sorted out the conflicts now.